### ISSUE TITLE
CET Overlay colors broken on HDR10 PQ mode

### ISSUE THREAD

**Issue by revanwolf** [maximegmd/CyberEngineTweaks#982]
# Bug

Operating system: Windows 10
Game version: 2.2
CET version: > 1.34.0 and 1.34.1
GPU: RX 6700 XT

## Description

On HDR 10 PQ mode the colors are totally broken on the last version

## Reproduction
Use the HDR10 PQ mode in the video parameters on the game

![Cyberpunk 2077 (C) 2020 by CD Projekt RED 13_12_2024 09_25_26](https://github.com/user-attachments/assets/4d0fb0d1-1596-4dcd-a1d3-d7137c88ab8d)

**Comment by WSSDude** [comment]
HDR was never properly supported with CET, there is also #955 which I suppose used different profile as they mention washed out colors... And bunch that are closed so we dont have 100 HDR issues opened...

Unsure if it will be fixed tbh, ImGui has ton of HDR issues open themselves, is just not well supportrd by the library.

**Comment by Rool-IO** [comment]
It's rather oversaturated than washed out in #955 but the root cause seems to be the same indeed.
If only just a workaround, lowering the alpha values would help a ton because the texts are eye-burning bright

**Comment by revanwolf** [comment]
Lowering the global alpha values to even 0.01 seems to fix it but i don't see a way to save this settings.

**Comment by WSSDude** [comment]
Yeah, wanted to add some themenig but it never got in... Removed the branch after some time, the style editor is only temporary and probably useful only to modders to find right values.
